src/aioswitcher/api/__init__.py

```diff
@@ -719,9 +779,21 @@ async def stop_shutter(self) -> SwitcherBaseResponse:
             "logged in session_id=%s, timestamp=%s", login_resp.session_id, timestamp
         )
 
-        packet = packets.RUNNER_STOP_COMMAND.format(
-            login_resp.session_id, timestamp, self._device_id
-        )
+        if self._device_type and self._device_type.token_needed and bool(self._token):
```

Comment:
This check is redundant, when you create the class you already checked for the conditions for the token, here just use the token if it is already set or skip the token if it is not set

Reply:
it is done.